lnst/Recipes/ENRT/BondRecipe.py

```diff
@@ -245,3 +246,7 @@ def pause_frames_dev_list(self):
         """
         return [self.matched.host1.eth0, self.matched.host1.eth1,
             self.matched.host2.eth0]
+
+    @property
+    def vf_trust_device_list(self) -> list[RemoteDevice]:
+        return [sriov_devices.phys_dev for sriov_devices in self.vf_config[self.matched.host1]]
```

Comment:
i think in this case the `sriov_devices` is a `list[Device]` and the `phys_dev` is a `Device` property so you need to nest this one more level:
```
return [device.phys_dev for devices in self.vf_config[self.matched.host1] for device in devices]
```
similar to other recipes...

and probably adjust the variable naming so it's consistent

Reply:
No, this is not necessary. SriovDevices is a collection of VFs over **single** PF. So iterating over devices in sriov_devices would generate several PFs that are identical.

Reply:
And `SriovDevices` class exports the `phys_dev` property that is common to all VFs in the class.

Reply:
Updated per https://github.com/LNST-project/lnst/pull/381#discussion_r1805965954